README.md

```diff
@@ -536,6 +536,44 @@ process.on('uncaughtException', async () => {
 });
 ```
 
+#### Queue#removeJob(jobId, [cb])
+
+```js
+queue.removeJob(3, function (err) {
+  if (!err) {
+    console.log('Job 3 was removed');
+  }
+});
+
+queue.removeJob(3)
+  .then(() => console.log('Job 3 was removed'));
+```
+
+Removes a job by its `jobId`. Idempotent.
+
+This may have unintended side-effect, e.g. if the job is currently being processed by another worker, so only use this method when you know it's safe.
+
+Returns the `Queue` instance it was called on.
+
+#### Queue#destroy([cb])
+
+```js
+queue.destroy(function (err) {
+  if (!err) {
+    console.log('Queue was destroyed');
+  }
+});
+
+queue.destroy()
+  .then(() => console.log('Queue was destroyed'));
+```
+
+Removes all Redis keys belonging to this queue, except `bq:name:events` and `bq:name:earlierDelayed` (see [Under the hood](#under-the-hood)). Idempotent.
```

Comment:
I wouldn't really worry about mentioning this - those are pubsub channel names, not keys.